python/GafferImageUI/ImageViewUI.py

```diff
@@ -1158,6 +1166,11 @@ def __init__( self, plug, **kw ) :
 
 		self._updateFromPlug()
 
+		self.dragEnterSignal().connectFront( Gaffer.WeakMethod( self.__dragEnter ), scoped = False )
+		self.dragLeaveSignal().connectFront( Gaffer.WeakMethod( self.__dragLeave ), scoped = False )
+		self.dropSignal().connectFront( Gaffer.WeakMethod( self.__drop ), scoped = False )
```

Comment:
I presume `connectFront()` is needed so that we get in before PlugValueWidget's own handlers? If so, could we comment that - it took me a little while to figure it out.

Reply:
I had actually copied the connectFront() carelessly from somewhere else that uses dragEnterSignal when I was just trying to get it working.  It's not absolutely necessary to get this working ( since the drags from nodes in the graph don't originate from PlugValueWidgets, PlugValueWidgets's handler doesn't grab them ).  But I think in light of your comment about returning True from  __dragEnter, I think it sounds like we should do a connectFront here, and I'll add a comment to reflect that?